tgarchive/example/template.html

```diff
@@ -14,16 +15,16 @@
 	<meta property="og:description" content="{{ meta_description }}" />
 	<meta property="og:type" content="website" />
 	<meta property="og:url" content="{{ config.site_url }}/.html" />
-	<meta property="og:image" content="{{ config.site_url }}/static/thumb.png" />
+	<meta property="og:image" content="{{ config.site_url }}/static/thumb.png" />{%
```

Comment:
Joining lines to remove indentation in generated HTML just makes the template difficult to read. Not-so-great indentation in automatically generated HTML is fine. Keeping the template indented and readable on the other hand, is necessary.